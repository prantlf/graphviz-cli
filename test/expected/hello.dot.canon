digraph G {
	graph [bb="0 0 101 150",
		label=Example
	];
	node [color=blue,
		label="\N",
		shape=Mdiamond
	];
	edge [color=red,
		label=Say
	];
	Hello -> World;
}
