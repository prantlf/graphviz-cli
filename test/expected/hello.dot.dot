digraph G {
	graph [bb="0,0,100.99,149.6",
		label=Example,
		lheight=0.23,
		lp="50.497,12.4",
		lwidth=0.69
	];
	node [color=blue,
		label="\N",
		shape=Mdiamond
	];
	edge [color=red,
		label=Say
	];
	Hello	[height=0.5,
		pos="50.497,131.6",
		width=1.2764];
	World	[height=0.5,
		pos="50.497,42.8",
		width=1.4027];
	Hello -> World	[lp="60.997,87.2",
		pos="e,50.497,60.913 50.497,113.45 50.497,101.32 50.497,84.819 50.497,70.919"];
}
