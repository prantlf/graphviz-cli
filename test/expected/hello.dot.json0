{
  "name": "G",
  "directed": true,
  "strict": false,
  "bb": "0,0,100.99,149.6",
  "label": "Example",
  "lheight": "0.23",
  "lp": "50.497,12.4",
  "lwidth": "0.69",
  "_subgraph_cnt": 0,
  "objects": [
    {
      "_gvid": 0,
      "name": "Hello",
      "color": "blue",
      "height": "0.5",
      "label": "\\N",
      "pos": "50.497,131.6",
      "shape": "Mdiamond",
      "width": "1.2763"
    },
    {
      "_gvid": 1,
      "name": "World",
      "color": "blue",
      "height": "0.5",
      "label": "\\N",
      "pos": "50.497,42.8",
      "shape": "Mdiamond",
      "width": "1.4027"
    }
  ],
  "edges": [
    {
      "_gvid": 0,
      "tail": 0,
      "head": 1,
      "color": "red",
      "label": "Say",
      "lp": "60.996,87.2",
      "pos": "e,50.497,60.913 50.497,113.45 50.497,101.32 50.497,84.819 50.497,70.919"
    }
  ]
}
