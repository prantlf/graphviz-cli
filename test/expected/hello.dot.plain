graph 1 1.4027 2.0778
node Hello 0.70135 1.8278 1.2764 0.5 Hello solid Mdiamond blue blue
node World 0.70135 0.59444 1.4027 0.5 World solid Mdiamond blue blue
edge Hello World 4 0.70135 1.5757 0.70135 1.4073 0.70135 1.178 0.70135 0.98499 Say 0.84718 1.2111 solid red
stop
