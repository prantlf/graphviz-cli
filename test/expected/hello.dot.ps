%!PS-Adobe-3.0
%%Creator: graphviz version 5.0.0 (20220707.1540)
%%Title: G
%%Pages: (atend)
%%BoundingBox: (atend)
%%EndComments
save
%%BeginProlog
/DotDict 200 dict def
DotDict begin

/setupLatin1 {
mark
/EncodingVector 256 array def
 EncodingVector 0

ISOLatin1Encoding 0 255 getinterval putinterval
EncodingVector 45 /hyphen put

% Set up ISO Latin 1 character encoding
/starnetISO {
        dup dup findfont dup length dict begin
        { 1 index /FID ne { def }{ pop pop } ifelse
        } forall
        /Encoding EncodingVector def
        currentdict end definefont
} def
/Times-Roman starnetISO def
/Times-Italic starnetISO def
/Times-Bold starnetISO def
/Times-BoldItalic starnetISO def
/Helvetica starnetISO def
/Helvetica-Oblique starnetISO def
/Helvetica-Bold starnetISO def
/Helvetica-BoldOblique starnetISO def
/Courier starnetISO def
/Courier-Oblique starnetISO def
/Courier-Bold starnetISO def
/Courier-BoldOblique starnetISO def
cleartomark
} bind def

%%BeginResource: procset graphviz 0 0
/coord-font-family /Times-Roman def
/default-font-family /Times-Roman def
/coordfont coord-font-family findfont 8 scalefont def

/InvScaleFactor 1.0 def
/set_scale {
       dup 1 exch div /InvScaleFactor exch def
       scale
} bind def

% styles
/solid { [] 0 setdash } bind def
/dashed { [9 InvScaleFactor mul dup ] 0 setdash } bind def
/dotted { [1 InvScaleFactor mul 6 InvScaleFactor mul] 0 setdash } bind def
/invis {/fill {newpath} def /stroke {newpath} def /show {pop newpath} def} bind def
/bold { 2 setlinewidth } bind def
/filled { } bind def
/unfilled { } bind def
/rounded { } bind def
/diagonals { } bind def
/tapered { } bind def

% hooks for setting color 
/nodecolor { sethsbcolor } bind def
/edgecolor { sethsbcolor } bind def
/graphcolor { sethsbcolor } bind def
/nopcolor {pop pop pop} bind def

/beginpage {	% i j npages
	/npages exch def
	/j exch def
	/i exch def
	/str 10 string def
	npages 1 gt {
		gsave
			coordfont setfont
			0 0 moveto
			(\() show i str cvs show (,) show j str cvs show (\)) show
		grestore
	} if
} bind def

/set_font {
	findfont exch
	scalefont setfont
} def

% draw text fitted to its expected width
/alignedtext {			% width text
	/text exch def
	/width exch def
	gsave
		width 0 gt {
			[] 0 setdash
			text stringwidth pop width exch sub text length div 0 text ashow
		} if
	grestore
} def

/boxprim {				% xcorner ycorner xsize ysize
		4 2 roll
		moveto
		2 copy
		exch 0 rlineto
		0 exch rlineto
		pop neg 0 rlineto
		closepath
} bind def

/ellipse_path {
	/ry exch def
	/rx exch def
	/y exch def
	/x exch def
	matrix currentmatrix
	newpath
	x y translate
	rx ry scale
	0 0 1 0 360 arc
	setmatrix
} bind def

/endpage { showpage } bind def
/showpage { } def

/layercolorseq
	[	% layer color sequence - darkest to lightest
		[0 0 0]
		[.2 .8 .8]
		[.4 .8 .8]
		[.6 .8 .8]
		[.8 .8 .8]
	]
def

/layerlen layercolorseq length def

/setlayer {/maxlayer exch def /curlayer exch def
	layercolorseq curlayer 1 sub layerlen mod get
	aload pop sethsbcolor
	/nodecolor {nopcolor} def
	/edgecolor {nopcolor} def
	/graphcolor {nopcolor} def
} bind def

/onlayer { curlayer ne {invis} if } def

/onlayers {
	/myupper exch def
	/mylower exch def
	curlayer mylower lt
	curlayer myupper gt
	or
	{invis} if
} def

/curlayer 0 def

%%EndResource
%%EndProlog
%%BeginSetup
14 default-font-family set_font
% /arrowlength 10 def
% /arrowwidth 5 def

% make sure pdfmark is harmless for PS-interpreters other than Distiller
/pdfmark where {pop} {userdict /pdfmark /cleartomark load put} ifelse
% make '<<' and '>>' safe on PS Level 1 devices
/languagelevel where {pop languagelevel}{1} ifelse
2 lt {
    userdict (<<) cvn ([) cvn load put
    userdict (>>) cvn ([) cvn load put
} if

%%EndSetup
setupLatin1
%%Page: 1 1
%%PageBoundingBox: 36 36 202 142
%%PageOrientation: Portrait
0 0 1 beginpage
gsave
36 36 166 106 boxprim clip newpath
1 1 set_scale 0 rotate 40 40 translate
0 0 0 graphcolor
14 /Times-Roman set_font
54.2296 8.2 moveto 49.76 (Example) alignedtext
% Hello
gsave
1 setlinewidth
0.66667 1 1 nodecolor
newpath 45.9494 60.8 moveto
.0505 42.8 lineto
45.9494 24.8 lineto
91.8483 42.8 lineto
closepath stroke
1 setlinewidth
0.66667 1 1 nodecolor
newpath 11.2222 47.1811 moveto
11.2222 38.4189 lineto
stroke
1 setlinewidth
0.66667 1 1 nodecolor
newpath 34.7778 29.1811 moveto
57.1211 29.1811 lineto
stroke
1 setlinewidth
0.66667 1 1 nodecolor
newpath 80.6767 38.4189 moveto
80.6767 47.1811 lineto
stroke
1 setlinewidth
0.66667 1 1 nodecolor
newpath 57.1211 56.4189 moveto
34.7778 56.4189 lineto
stroke
0 0 0 nodecolor
14 /Times-Roman set_font
30.3977 38.6 moveto 31.1 (Hello) alignedtext
grestore
% World
gsave
1 setlinewidth
0.66667 1 1 nodecolor
newpath 107.7208 97.7905 moveto
57.2262 79.7905 lineto
107.7208 61.7905 lineto
158.2154 79.7905 lineto
closepath stroke
1 setlinewidth
0.66667 1 1 nodecolor
newpath 68.5295 83.8199 moveto
68.5295 75.7612 lineto
stroke
1 setlinewidth
0.66667 1 1 nodecolor
newpath 96.4175 65.8199 moveto
119.0241 65.8199 lineto
stroke
1 setlinewidth
0.66667 1 1 nodecolor
newpath 146.9121 75.7612 moveto
146.9121 83.8199 lineto
stroke
1 setlinewidth
0.66667 1 1 nodecolor
newpath 119.0241 93.7612 moveto
96.4175 93.7612 lineto
stroke
0 0 0 nodecolor
14 /Times-Roman set_font
89.838 75.5905 moveto 35.77 (World) alignedtext
grestore
% Hello->World
gsave
1 setlinewidth
0 1 1 edgecolor
newpath 64.1488 53.6983 moveto
69.0732 56.6472 74.5185 59.908 79.8434 63.0967 curveto
stroke
0 1 1 edgecolor
newpath 78.2003 66.1923 moveto
88.5779 68.3272 lineto
81.7967 60.1868 lineto
closepath fill
1 setlinewidth
solid
0 1 1 edgecolor
newpath 78.2003 66.1923 moveto
88.5779 68.3272 lineto
81.7967 60.1868 lineto
closepath stroke
0 0 0 edgecolor
14 /Times-Roman set_font
71.9961 45.7975 moveto 21 (Say) alignedtext
grestore
endpage
showpage
grestore
%%PageTrailer
%%EndPage: 1
%%Trailer
%%Pages: 1
%%BoundingBox: 36 36 202 142
end
restore
%%EOF
