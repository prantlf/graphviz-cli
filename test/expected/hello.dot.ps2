%!PS-Adobe-3.0
%%Creator: graphviz version 2.44.0 (20200408.0750)
%%Title: G
%%Pages: (atend)
%%BoundingBox: (atend)
%%EndComments
save
%%BeginProlog
/DotDict 200 dict def
DotDict begin

/setupLatin1 {
mark
/EncodingVector 256 array def
 EncodingVector 0

ISOLatin1Encoding 0 255 getinterval putinterval
EncodingVector 45 /hyphen put

% Set up ISO Latin 1 character encoding
/starnetISO {
        dup dup findfont dup length dict begin
        { 1 index /FID ne { def }{ pop pop } ifelse
        } forall
        /Encoding EncodingVector def
        currentdict end definefont
} def
/Times-Roman starnetISO def
/Times-Italic starnetISO def
/Times-Bold starnetISO def
/Times-BoldItalic starnetISO def
/Helvetica starnetISO def
/Helvetica-Oblique starnetISO def
/Helvetica-Bold starnetISO def
/Helvetica-BoldOblique starnetISO def
/Courier starnetISO def
/Courier-Oblique starnetISO def
/Courier-Bold starnetISO def
/Courier-BoldOblique starnetISO def
cleartomark
} bind def

%%BeginResource: procset graphviz 0 0
/coord-font-family /Times-Roman def
/default-font-family /Times-Roman def
/coordfont coord-font-family findfont 8 scalefont def

/InvScaleFactor 1.0 def
/set_scale {
       dup 1 exch div /InvScaleFactor exch def
       scale
} bind def

% styles
/solid { [] 0 setdash } bind def
/dashed { [9 InvScaleFactor mul dup ] 0 setdash } bind def
/dotted { [1 InvScaleFactor mul 6 InvScaleFactor mul] 0 setdash } bind def
/invis {/fill {newpath} def /stroke {newpath} def /show {pop newpath} def} bind def
/bold { 2 setlinewidth } bind def
/filled { } bind def
/unfilled { } bind def
/rounded { } bind def
/diagonals { } bind def
/tapered { } bind def

% hooks for setting color 
/nodecolor { sethsbcolor } bind def
/edgecolor { sethsbcolor } bind def
/graphcolor { sethsbcolor } bind def
/nopcolor {pop pop pop} bind def

/beginpage {	% i j npages
	/npages exch def
	/j exch def
	/i exch def
	/str 10 string def
	npages 1 gt {
		gsave
			coordfont setfont
			0 0 moveto
			(\() show i str cvs show (,) show j str cvs show (\)) show
		grestore
	} if
} bind def

/set_font {
	findfont exch
	scalefont setfont
} def

% draw text fitted to its expected width
/alignedtext {			% width text
	/text exch def
	/width exch def
	gsave
		width 0 gt {
			[] 0 setdash
			text stringwidth pop width exch sub text length div 0 text ashow
		} if
	grestore
} def

/boxprim {				% xcorner ycorner xsize ysize
		4 2 roll
		moveto
		2 copy
		exch 0 rlineto
		0 exch rlineto
		pop neg 0 rlineto
		closepath
} bind def

/ellipse_path {
	/ry exch def
	/rx exch def
	/y exch def
	/x exch def
	matrix currentmatrix
	newpath
	x y translate
	rx ry scale
	0 0 1 0 360 arc
	setmatrix
} bind def

/endpage { showpage } bind def
/showpage { } def

/layercolorseq
	[	% layer color sequence - darkest to lightest
		[0 0 0]
		[.2 .8 .8]
		[.4 .8 .8]
		[.6 .8 .8]
		[.8 .8 .8]
	]
def

/layerlen layercolorseq length def

/setlayer {/maxlayer exch def /curlayer exch def
	layercolorseq curlayer 1 sub layerlen mod get
	aload pop sethsbcolor
	/nodecolor {nopcolor} def
	/edgecolor {nopcolor} def
	/graphcolor {nopcolor} def
} bind def

/onlayer { curlayer ne {invis} if } def

/onlayers {
	/myupper exch def
	/mylower exch def
	curlayer mylower lt
	curlayer myupper gt
	or
	{invis} if
} def

/curlayer 0 def

%%EndResource
%%EndProlog
%%BeginSetup
14 default-font-family set_font
% /arrowlength 10 def
% /arrowwidth 5 def

% make sure pdfmark is harmless for PS-interpreters other than Distiller
/pdfmark where {pop} {userdict /pdfmark /cleartomark load put} ifelse
% make '<<' and '>>' safe on PS Level 1 devices
/languagelevel where {pop languagelevel}{1} ifelse
2 lt {
    userdict (<<) cvn ([) cvn load put
    userdict (>>) cvn ([) cvn load put
} if

%%EndSetup
setupLatin1
%%Page: 1 1
%%PageBoundingBox: 36 36 145 194
%%PageOrientation: Portrait
<< /PageSize [145 194] >> setpagedevice
0 0 1 beginpage
gsave
36 36 109 158 boxprim clip newpath
1 1 set_scale 0 rotate 40 -189.6 translate
[ /CropBox [36 36 145 194] /PAGES pdfmark
0 0 0 graphcolor
14 /Times-Roman set_font
25.616 8.2 moveto 49.76 (Example) alignedtext
% Hello
gsave
1 setlinewidth
0.66667 1 1 nodecolor
newpath 50.4975 149.6 moveto
4.6008 131.6 lineto
50.4975 113.6 lineto
96.3942 131.6 lineto
closepath stroke
1 setlinewidth
0.66667 1 1 nodecolor
newpath 15.7723 135.9813 moveto
15.7723 127.2187 lineto
stroke
1 setlinewidth
0.66667 1 1 nodecolor
newpath 39.3259 117.9813 moveto
61.669 117.9813 lineto
stroke
1 setlinewidth
0.66667 1 1 nodecolor
newpath 85.2226 127.2187 moveto
85.2226 135.9813 lineto
stroke
1 setlinewidth
0.66667 1 1 nodecolor
newpath 61.669 145.2187 moveto
39.3259 145.2187 lineto
stroke
0 0 0 nodecolor
14 /Times-Roman set_font
34.9463 127.4 moveto 31.1 (Hello) alignedtext
grestore
% World
gsave
1 setlinewidth
0.66667 1 1 nodecolor
newpath 50.4975 60.8 moveto
.0025 42.8 lineto
50.4975 24.8 lineto
100.9924 42.8 lineto
closepath stroke
1 setlinewidth
0.66667 1 1 nodecolor
newpath 11.3058 46.8293 moveto
11.3058 38.7707 lineto
stroke
1 setlinewidth
0.66667 1 1 nodecolor
newpath 39.1942 28.8293 moveto
61.8008 28.8293 lineto
stroke
1 setlinewidth
0.66667 1 1 nodecolor
newpath 89.6891 38.7707 moveto
89.6891 46.8293 lineto
stroke
1 setlinewidth
0.66667 1 1 nodecolor
newpath 61.8008 56.7707 moveto
39.1942 56.7707 lineto
stroke
0 0 0 nodecolor
14 /Times-Roman set_font
32.6146 38.6 moveto 35.77 (World) alignedtext
grestore
% Hello->World
gsave
1 setlinewidth
0 1 1 edgecolor
newpath 50.4975 113.4511 moveto
50.4975 101.3249 50.4975 84.819 50.4975 70.919 curveto
stroke
0 1 1 edgecolor
newpath 53.9976 70.9132 moveto
50.4975 60.9133 lineto
46.9976 70.9133 lineto
closepath fill
1 setlinewidth
solid
0 1 1 edgecolor
newpath 53.9976 70.9132 moveto
50.4975 60.9133 lineto
46.9976 70.9133 lineto
closepath stroke
0 0 0 edgecolor
14 /Times-Roman set_font
50.4975 83 moveto 21 (Say) alignedtext
grestore
endpage
showpage
grestore
%%PageTrailer
%%EndPage: 1
%%Trailer
%%Pages: 1
%%BoundingBox: 36 36 145 194
end
restore
%%EOF
