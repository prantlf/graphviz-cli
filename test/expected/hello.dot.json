{
  "name": "G",
  "directed": true,
  "strict": false,
  "_draw_": 
  [
    {
      "op": "c",
      "grad": "none",
      "color": "#fffffe00"
    },
    {
      "op": "C",
      "grad": "none",
      "color": "#ffffff"
    },
    {
      "op": "P",
      "points": [[0.000,0.000],[0.000,149.600],[100.990,149.600],[100.990,0.000]]
    }
  ],
  "_ldraw_": 
  [
    {
      "op": "F",
      "size": 14.000,
      "face": "Times-Roman"
    },
    {
      "op": "c",
      "grad": "none",
      "color": "#000000"
    },
    {
      "op": "T",
      "pt": [50.500,8.200],
      "align": "c",
      "width": 49.760,
      "text": "Example"
    }
  ],
  "bb": "0,0,100.99,149.6",
  "label": "Example",
  "lheight": "0.23",
  "lp": "50.497,12.4",
  "lwidth": "0.69",
  "xdotversion": "1.7",
  "_subgraph_cnt": 0,
  "objects": [
    {
      "_gvid": 0,
      "name": "Hello",
      "_draw_": 
      [
        {
          "op": "c",
          "grad": "none",
          "color": "#0000ff"
        },
        {
          "op": "p",
          "points": [[50.500,149.600],[4.600,131.600],[50.500,113.600],[96.400,131.600]]
        },
        {
          "op": "c",
          "grad": "none",
          "color": "#0000ff"
        },
        {
          "op": "L",
          "points": [[15.770,135.980],[15.770,127.220]]
        },
        {
          "op": "c",
          "grad": "none",
          "color": "#0000ff"
        },
        {
          "op": "L",
          "points": [[39.330,117.980],[61.670,117.980]]
        },
        {
          "op": "c",
          "grad": "none",
          "color": "#0000ff"
        },
        {
          "op": "L",
          "points": [[85.220,127.220],[85.220,135.980]]
        },
        {
          "op": "c",
          "grad": "none",
          "color": "#0000ff"
        },
        {
          "op": "L",
          "points": [[61.670,145.220],[39.330,145.220]]
        }
      ],
      "_ldraw_": 
      [
        {
          "op": "F",
          "size": 14.000,
          "face": "Times-Roman"
        },
        {
          "op": "c",
          "grad": "none",
          "color": "#000000"
        },
        {
          "op": "T",
          "pt": [50.500,127.400],
          "align": "c",
          "width": 31.100,
          "text": "Hello"
        }
      ],
      "color": "blue",
      "height": "0.5",
      "label": "\\N",
      "pos": "50.497,131.6",
      "shape": "Mdiamond",
      "width": "1.2764"
    },
    {
      "_gvid": 1,
      "name": "World",
      "_draw_": 
      [
        {
          "op": "c",
          "grad": "none",
          "color": "#0000ff"
        },
        {
          "op": "p",
          "points": [[50.500,60.800],[0.000,42.800],[50.500,24.800],[100.990,42.800]]
        },
        {
          "op": "c",
          "grad": "none",
          "color": "#0000ff"
        },
        {
          "op": "L",
          "points": [[11.310,46.830],[11.310,38.770]]
        },
        {
          "op": "c",
          "grad": "none",
          "color": "#0000ff"
        },
        {
          "op": "L",
          "points": [[39.190,28.830],[61.800,28.830]]
        },
        {
          "op": "c",
          "grad": "none",
          "color": "#0000ff"
        },
        {
          "op": "L",
          "points": [[89.690,38.770],[89.690,46.830]]
        },
        {
          "op": "c",
          "grad": "none",
          "color": "#0000ff"
        },
        {
          "op": "L",
          "points": [[61.800,56.770],[39.190,56.770]]
        }
      ],
      "_ldraw_": 
      [
        {
          "op": "F",
          "size": 14.000,
          "face": "Times-Roman"
        },
        {
          "op": "c",
          "grad": "none",
          "color": "#000000"
        },
        {
          "op": "T",
          "pt": [50.500,38.600],
          "align": "c",
          "width": 35.770,
          "text": "World"
        }
      ],
      "color": "blue",
      "height": "0.5",
      "label": "\\N",
      "pos": "50.497,42.8",
      "shape": "Mdiamond",
      "width": "1.4027"
    }
  ],
  "edges": [
    {
      "_gvid": 0,
      "tail": 0,
      "head": 1,
      "_draw_": 
      [
        {
          "op": "c",
          "grad": "none",
          "color": "#ff0000"
        },
        {
          "op": "b",
          "points": [[50.500,113.450],[50.500,101.320],[50.500,84.820],[50.500,70.920]]
        }
      ],
      "_hdraw_": 
      [
        {
          "op": "S",
          "style": "solid"
        },
        {
          "op": "c",
          "grad": "none",
          "color": "#ff0000"
        },
        {
          "op": "C",
          "grad": "none",
          "color": "#ff0000"
        },
        {
          "op": "P",
          "points": [[54.000,70.910],[50.500,60.910],[47.000,70.910]]
        }
      ],
      "_ldraw_": 
      [
        {
          "op": "F",
          "size": 14.000,
          "face": "Times-Roman"
        },
        {
          "op": "c",
          "grad": "none",
          "color": "#000000"
        },
        {
          "op": "T",
          "pt": [61.000,83.000],
          "align": "c",
          "width": 21.000,
          "text": "Say"
        }
      ],
      "color": "red",
      "label": "Say",
      "lp": "60.997,87.2",
      "pos": "e,50.497,60.913 50.497,113.45 50.497,101.32 50.497,84.819 50.497,70.919"
    }
  ]
}
